sdcm/scan_operation_thread.py

```diff
@@ -376,14 +381,17 @@ def _compare_output_files(self) -> bool:
             file_names = f' {self.normal_query_output.name} {self.reversed_query_output.name}'
             diff_cmd = f"diff -y --suppress-common-lines {file_names}"
             self.log.debug("Comparing scan queries output files by: %s", diff_cmd)
-            result = LOCAL_CMD_RUNNER.run(cmd=diff_cmd, ignore_status=True)
+            log_file = Path(TestConfig().logdir()) / 'fullscans' / \
+                f'partition_range_scan_diff_{datetime.datetime.now(tz=utc).strftime("%Y_%m_%d-%I_%M_%S")}.log'
```

Comment:
'partition_range' -> 'partition_rows'

Reply:
You can open a PR, if you think it's important enough